x-pack/plugins/security_solution/scripts/junit_transformer/lib.ts

```diff
@@ -18,6 +18,20 @@ import { PathReporter } from 'io-ts/lib/PathReporter';
 import globby from 'globby';
 import del from 'del';
 
+// Function to remove specific fields from an XML object in order to
+// compare them as strings.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function removeFields(obj: any, fieldsToRemove: string[]): any {
```

Comment:
any reason for not having an explicit test for this function?

Reply:
Checking over the tests of the whole package `junit_transformer` it looks like it is not expicitly tested as it relates to test code. I guess its not needed to try to keep a non existing test coverage. Also this part of code is tested by the generic test `updates the file in place, applying the expected transformation` as all of the rest of the code.